src/OneExplorer.ts

```diff
@@ -106,9 +129,11 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<OneNode> {
       if (node.type === NodeType.directory) {
         return new OneNode(node.name, vscode.TreeItemCollapsibleState.Expanded, node);
       } else if (node.type === NodeType.model) {
+        return new OneNode(node.name, vscode.TreeItemCollapsibleState.None, node);
```

Comment:
Can we expand all nodes by default?
Every time when I finds cfg file, I have to expand tflite file by clicking the tflite file. By removing those clicking it would provide better usability.

Reply:
I have collapsed basemodel because most users have 10+ configs under their directory. 
Basically it expands all directory. It will be irritating to fold all everytime you open the explorer.
I made this choice based on `cmake` explorer use experience.